src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -146,6 +146,31 @@ public async Task<IActionResult> GetCurrentForecastData([FromQuery(Name = "locat
         }
     }
 
+    /// <summary>
+    /// Given an array of requested historical forecasts, retrieve the forecasted data and calculate the optimal
+    /// marginal carbon intensity window. 
+    /// </summary>
+    /// <param name="requestedForecasts"> Array of requested forecasts.</param>
+    /// <returns>An array of forecasts with their optimal marginal carbon intensity window.</returns>
+    /// <response code="200">Returns the requested forecast objects</response>
```

Comment:
🐑 since these are generated by swagger too, do we want to intentionally call each out instead of leaving the swagger? And if so, maybe add an item about quickly bringing all other calls up to date during sharing

Reply:
The reason to use both is for the descriptive comment.  Swagger defaults to the http status code description `200 Successful`, `400 Bad Request` etc.

I think bringing them all up to this standard during sharing would be nice.